src/domain/mainwindow.py

```diff
@@ -191,10 +191,8 @@ def run(self):
                 self.selected_operation_mode
                 == OperationMode.OperationModeTypes.TestModelMode
             ):
-                if not self.check_classification_model():
-                    logger.error(
-                        "Cannot run this mode without classification model. Aborting."
-                    )
+                if not self.check_models():
+                    logger.error("Cannot run this mode without AI models. Aborting.")
```

Comment:
this sounds more like a CRITICAL rather than an ERROR :) even though the program is still running, without AI working we cannot do much, so let's consider this as critical. Agree?

Reply:
I just copied the one that was already there. However, I believe that `error` it is most suited here as ciritical implies a termination of the application, as described on PEP: https://peps.python.org/pep-0282/#levels

Reply:
ok